src/resources/js/.eslintrc

```diff
@@ -27,5 +27,6 @@
     "tecTicketsCommerceData": true,
     "tecTicketsCommerceGatewayPayPalCheckout": true,
     "paypal": true
-  }
+  },
+  "ignorePatterns": ["**/*.test.js"]
```

Comment:
What is the reason for this? If it's for the `tickets-utils.test.js`, I'd suggest either fixing the lint issues, disabling the eslint for the specific rows where we can't fix it, or disabling it for the entire file if there are too many that you cannot fix.

Reply:
There were a lot of linting errors that were showing up. Many of them were just for using arrow functions. So, not sure what linting config I need to change.

I have disabled the ignore pattern and the errors show up in the lint action: https://github.com/the-events-calendar/event-tickets/runs/6485383345?check_suite_focus=true

Let me know what changes I may make around the config or code to resolve these. Thanks for the help.

Reply:
For the `no-undef` error on globals, see how it's done here: https://github.com/the-events-calendar/event-tickets/blob/fd5054e198c58a7d26d6e4a129ed976b4a891b4b/src/resources/js/tickets-attendees.js#L1
For the `es5/no-arrow-functions` error, it looks like the file you are testing is ES5? I'd say determine whether you write tests in ES5 or ES6 for files that are written in ES5, and if you do decide to continue writing tests in ES6, write on the top of the file `/* eslint-disable es5/no-arrow-functions */`.
Same thing with the `es5/no-destructuring` and `es5/no-shorthand-properties` errors.

Reply:
I tried adding the global comment previously as you have referred but that results in more errors of `no-redeclare` and `unused-vars`: https://d.pr/i/S6ZUFA

Now, I just ended up suppressing all the rules that came up which looks weird on top, but I am thinking it is much better to ignore the whole test file from the `.eslintrc` config instead.

The only confusion I now have is, why is it still triggering the `es5` rules? is it the `require` causing this?
I want to write tests only using `es6` as going forward, we are only going to follow `es6` on every new JS file.

Reply:
@paulmskim 
Also, the lint is failing for `async` keyword here: https://github.com/the-events-calendar/event-tickets/blob/07eb414c803b45f154449e74ff265b35f8ce9b64/src/resources/js/admin/gateway/stripe/webhooks.js#L102

I tried changing configs but I was not able to resolve this.

Reply:
The reason why you're getting these lint errors is because for `src/resources/js/**/*.js` files, it uses a different config from `src/modules/**/*.js` files. If the consensus is to move from es5 to es6 in `src/resources/js/**/*.js` files, then it may be worth changing the config files in product taskmaster so that it allows es6 syntax.

For the async stuff, you could probably tackle it in another PR, since it doesn't really affect this task.